gap/Algebroids.gd

```diff
@@ -146,6 +146,20 @@ DeclareOperation( "AssignSetOfObjects",
 DeclareAttribute( "SetOfGeneratingMorphisms",
         IsAlgebroid );
 
+#! @Description
+#!  The subset of the generating morphisms that starts at <A>obj_1</A> and ends at <A>obj_2</A>.
+#! @Arguments A, obj_1, obj_2
+#! @Returns a list
+DeclareOperation( "SetOfGeneratingMorphisms",
```

Comment:
Could you please provide a convenience method only taking the two objects, extracting the algebroid from the first, and then call the three-argument version. It would also be nice if you could provide a test in one of the examples files for the code coverage to pass.

Reply:
I have already pushed the new changes.